src/mix_all.c

```diff
@@ -170,7 +170,6 @@
     int old_vr = vi->old_vr
 
 #ifndef LIBXMP_CORE_DISABLE_IT
-
```

Comment:
See above comments.

Reply:
Dropping changes to mix_all.c in next force-push